docs/concepts/flow/01-overview.md

```diff
@@ -0,0 +1,79 @@
+linear and can be defined as:
+
+```math
+\text{amount owed} = rps \cdot \text{elapsed time}
+```
+
+Flow protocol can be used in several areas of everyday finance, such as payroll, distributing grants, insurance
+premiums, loans interest, token ESOPs etc. If you are looking for vesting and airdrops, please refer to our
+[Lockup](../lockup/overview) protocol.
+
+## Features
+
+1. **Flexible deposit:** A stream can be funded with any amount, at any time, by anyone, in full or in parts.
+2. **Flexible duration:** A stream can be created with no specific start or end time. It can run indefinitely.
+3. **Pause:** A stream can be paused by the sender and can later be restarted without losing track of previously accrued
+   debt.
+4. **Refund:** Unstreamed amount can be refunded back to the sender at any time.
+5. **Void:** Voiding a stream implies it cannot be restarted anymore. Voiding an insolvent stream forfeits the uncovered
+   debt. Either party can void a stream at any time.
+6. **Withdraw:** it is publicly callable as long as `to` is set to the recipient. However, a stream’s recipient is
+   allowed to withdraw funds to any address.
+
+## Key Definitions
+
+The definitions below will help you understand some terms used in Sablier Flow:
+
+### Stream balance
+
+Stream balance is the token balance of a stream. It increases when funds are deposited into a stream, and decreases when
+the sender refunds from it or when a withdrawal happens.
+
+### Total debt
+
+Total debt is the amount of tokens owed to the recipient. This value is further divided into two sub-categories:
+
+- **Covered debt:** The part of the total debt that covered by the stream balance. This is the same as the
+  **withdrawable amount**, which is an alias.
+- **Uncovered debt:** The part of the total debt that is not covered by the stream balance. This is what the sender owes
+  to the stream.
+
+```math
+\text{total debt} = \text{covered debt} + \text{uncovered debt}
+```
+
+### Snapshot debt and Snapshot time
+
+A snapshot is an event during which snapshot debt and snapshot time of a Flow stream are updated. **Snapshot debt** is
+the debt accumulated since the previous snapshot. The UNIX timestamp at which snapshot debt is updated is called
+**Snapshot time**.
+
+At snapshot, the following calculations are taking place:
+
+```math
+\text{snapshot debt} = \text{previous snapshot debt} + \underbrace{
+rps \cdot (\text{block.timestamp} - \text{snapshot time})}_\text{ongoing debt}
+\text{snapshot time} = \text{block.timestamp}
+```
+
+Therefore, at any point in time, total debt can also be defined as:
+
+```math
+\text{total debt} = \text{snapshot debt} + \text{ongoing debt}
+```
+
+## Lifecycle
+
+1. A Flow stream is created with an `rps`, a `sender` and a `recipient` address.
+2. During the lifecycle of the stream, all the functions enclosed inside the dotted rectangle (diagram below) can be
```

Comment:
I've added a new diagram through https://github.com/sablier-labs/docs/pull/200. You can see the [preview here](https://v2-docs-git-feat-sequence-diagrams-sablier.vercel.app/reference/flow/diagrams#flow-actors-with-actions). Re [function calls](https://github.com/sablier-labs/flow/blob/378049ec74c0077c6ce62d47f2977acdd920ee40/DIAGRAMS.md#function-calls), I am not sure if it would be useful in the docs or not. 

What do you think?